src/scripts/scan.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+docker run -it \
+  --env MAPI_TOKEN \
+  --env GITHUB_TOKEN \
+  --env POSTMAN_ENVIRONMENT \
+  --env POSTMAN_API_KEY \
+  forallsecure/mapi:latest \
```

Comment:
Do we need to support APIs that are running locally in a separate docker network. ie: using the `--network` flag?